packages/client/src/context/UIComponents/styles.component.tsx

```diff
@@ -0,0 +1,59 @@
+// Styles for signup/login/resetpassword page
+const styles = {
```

Comment:
For this project, since we are leveraging MUI, the avoid having additional CSS styling elsewhere. MUI handles 90% of the styling needs out of the box. Another 5% is handled by the theme style which keeps the styling consistent across the website, and the last 5% is applied directly to the MUI component.